scripts/complete_liquidity_provision.js

```diff
@@ -92,11 +94,18 @@ module.exports = async callback => {
         callback("Error: Bound checks did not pass")
       }
     }
+    if (argv.fleetSize > 23) {
+      callback("Error: Choose a smaller fleetSize, otherwise your payload will be to big for Metamask")
+    }
 
     console.log(`2. Deploying ${argv.fleetSize} trading brackets`)
     const bracketAddresses = await deployFleetOfSafes(masterSafe.address, argv.fleetSize, true)
     console.log("Following bracket-traders have been deployed", bracketAddresses.join())
 
+    // Sleeping for 5 seconds to make sure Metamask has processed all new deployed contracts so that
```

Comment:
Also this is a workaround because of an issue with Infura, so I'd say that in the comment, Metamask is innocent.